rotkehlchen/chain/evm/frontend_structures/types.py

```diff
@@ -0,0 +1,46 @@
+from enum import auto
+from typing import Literal, NamedTuple, Optional
+
+from rotkehlchen.utils.mixins.serializableenum import SerializableEnumMixin
+
+
+class ProtocolDetails(NamedTuple):
+    identifier: str
+    label: str
+    image: str
+
+
+class EventDetails(NamedTuple):
+    label: str
+    icon: str
+    color: Optional[Literal['green', 'red']] = None
+
+
+class TransactionEventType(SerializableEnumMixin):
```

Comment:
And tell frontend to also change their shit. But also why duplicate? Can't we just export this enum to them? They should not duplicate this data.

```suggestion
class EventCategory(SerializableEnumMixin):
```